backend/src/main/java/org/sefglobal/core/controller/MultiverseController.java

```diff
@@ -62,8 +66,11 @@ public University updateUniversity(@PathVariable long id,
     //todo: Implement an endpoint to delete university (change active status)
 
     @GetMapping("/ambassadors")
-    public Iterable<Ambassador> listAmbassador() {
-        return ambassadorRepository.findAll();
+    public Iterable<Ambassador> listAmbassador(@RequestParam(defaultValue = "0",
+            required = false) int pageNumber,
```

Comment:
Change the name to `limit`

Reply:
but it works like a page number

Reply:
Oh sorry my mistake I have commented on the wrong line.

Reply:
@anjula-sack we need to get the `limit` and `offset` from the user; not `pageNumber` and `offset`